README.md

```diff
@@ -1,74 +1,74 @@
 
-> Description the project.
+You will create a Ruby on Rails application that allows the user to:
 
+- register and log in, so that the data is private to them.
+- introduce new transactions associated with a category.
+- see the money spent on each category.
 
-## Built With
-
-- Ruby on Rails
+This app was created with Rails, including Devise for authentication.
 
+## Live Version
 
-## Live Demo (if available)
+[Live](https://sleepy-shelf-32739.herokuapp.com/)
 
-[Live Demo Link](https://livedemo.com)
+## Built With:
 
+- Ruby on Rails
+- RSpec
+- SASS
+- PostgresSQL
+- Devise
 
-## Getting Started
+Prerequisites
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+- Ruby 3.1.2 or newer
+- Rails 7.0.4 or newer
+- Git
+- NPM
+- PostgreSQL
+- Modern web browser
 
+## Getting Started
+- To get a local copy of this project clone this repository or download the Zip folder:
+git clone https://github.com/fibini/Budget_app
 
-To get a local copy up and running follow these simple example steps.
+- To get started, In the root directory run:
+gem install bundler
 
-### Prerequisites
+- Then run:
+bundle install
 
-### Setup
+- Run ruby on rails app in the Command Line:
+rails s
 
-### Install
+- Then browse http://localhost:3000
 
 ### Usage
 
-### Run tests
-
-### Deployment
-
-
-
-## Authors
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+To edit this project:
 
-👤 **Author2**
+- Open the repository you cloned with a code editor that you like.
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- Edit the code as you want.
 
-## 🤝 Contributing
+## Author:
 
-Contributions, issues, and feature requests are welcome!
+👤 **Fabien**
 
-Feel free to check the [issues page](../../issues/).
+- GitHub: [![GitHub Badge](https://img.shields.io/badge/-fibini-white?logo=GitHub&logoColor=181717&style=plastic)](https://github.com/fibini)
 
-## Show your support
+- Twitter: [![Twitter Badge](https://img.shields.io/badge/-Onenewpage1-white?logo=Twitter&logoColor=1DA1F2&style=plastic)](https://twitter.com/Onenewpage1)
 
-Give a ⭐️ if you like this project!
+- LinkedIn: [![LinkedIn Badge](https://img.shields.io/badge/-FabienBrathwaite-white?logo=LinkedIn&logoColor=1DA1F2&style=plastic)](https://www.linkedin.com/in/fabien-brathwaite/)
 
-## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+## :star2: Show your support
```

Comment:
- [ ]  you should add credit to the author of the design as stated in the requirements 
<img width="1107" alt="Screen Shot 2022-10-14 at 19 52 51" src="https://user-images.githubusercontent.com/61790587/195920112-d75a2e72-795e-49f5-ae80-f10e673918bb.png">